tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/provider/JCloudBlobStoreProvider.java

```diff
@@ -279,47 +279,47 @@ public ProviderMetadata getProviderMetadata() {
     };
 
     static final CredentialBuilder AWS_CREDENTIAL_BUILDER = (TieredStorageConfiguration config) -> {
-        if (config.getCredentials() == null) {
-            final AWSCredentialsProvider authChain;
-            try {
-                if (Strings.isNullOrEmpty(config.getConfigProperty(S3_ROLE_FIELD))) {
-                    authChain = DefaultAWSCredentialsProviderChain.getInstance();
+//        if (config.getCredentials() == null) {
```

Comment:
Just a comment on style here: in a future PR, I recommend removing this line. Its record will be stored in git history, and there isn't any need to keep it for future reference.